ideascube/serveradmin/package.py

```diff
@@ -0,0 +1,57 @@
+
+
+import yaml
+import os.path as op
+import zipfile
+import codecs
+import csv
+
+
+class MediaCenterPackage:
+    def __init__(self, working_dir, medias=None):
+        self.working_dir = working_dir
+        self.medias = list(filter(self.check_media, medias or []))
+
+    def check_media(self, media):
+        for required_metadata in ('title', 'summary', 'credits', 'path'):
+            if not media.get(required_metadata):
+                return False
+        if not op.exists(op.join(self.working_dir, media['path'])):
+            return False
+        preview_path = media.get('preview')
+        if (preview_path
+         and not op.exists(op.join(self.working_dir, preview_path))):
+            print("Warning : preview_path {} defined but file does not exists"
+                  .format(preview_path))
+            media['preview'] = None
```

Comment:
I'm not sure `check_media` should alter the metadatas. I'd move the print when actually using the preview path (when creating the zip).